movie/src/components/Card/index.jsx

```diff
@@ -55,15 +73,27 @@ const Card = ({ movies }) => {
                 <Item>
                   <img src={IMAGE_URL} alt="" />
                 </Item>
-                <Item className="description">
-                  <Item className="voteAverage">{movie.vote_average}</Item>
+                <div className="description">
+                  <div className="voteAverage">
+                    <CircularProgress
+                      color={setCircleColor(
+                        voteAverageAsPercent(movie.vote_average)
```

Comment:
looks like you're keep using `voteAverageAsPercent(movie.vote_average)` so please create a const for it

Reply:
Yep, I got it.